FFXIVClientStructs/FFXIV/Component/GUI/AtkTextInput.cs

```diff
@@ -40,8 +40,19 @@ public unsafe partial struct AtkTextInput {
     [GenerateInterop(true)]
     [VirtualTable("48 89 86 ?? ?? ?? ?? 48 8D 05 ?? ?? ?? ?? 48 89 86 ?? ?? ?? ?? E8 ?? ?? ?? ?? 40 F6 C5 01", 10, 5)]
     [StructLayout(LayoutKind.Explicit, Size = 0x8)]
-    public unsafe partial struct AtkTextInputEventInterface {
+    public partial struct AtkTextInputEventInterface {
+        [VirtualFunction(1)]
+        public partial void UpdateTextSelection(TextSelectionInfo* selectionInfo);
+
         [VirtualFunction(4)]
         public partial AtkResNode* GetOwnerNode();
     }
+
+    [StructLayout(LayoutKind.Explicit, Size = 0x10)]
+    public struct TextSelectionInfo {
+        [FieldOffset(0x0)] public bool CharacterAdded;
+        [FieldOffset(0x4)] public ushort SelectionStart;
+        [FieldOffset(0x8)] public ushort SelectionEnd;
+        [FieldOffset(0xC)] public ushort StringLength;
+    }
```

Comment:
This structure is larger than `0x10` bytes due to it containing at least 2 Utf8String pointers one at `0x8` and the other at `0x10` 

This is the correct structure 
```suggestion
    [StructLayout(LayoutKind.Explicit, Size = 0x18)]
    public struct TextSelectionInfo {
        [FieldOffset(0x0)] public short CharactersAdded;
        [FieldOffset(0x2)] public ushort SelectionStart;
        [FieldOffset(0x4)] public ushort SelectionEnd;
        [FieldOffset(0x6)] public ushort StringLength;
        [FieldOffset(0x8)] public Utf8String* String1;
        [FieldOffset(0x10)] public Utf8String* String2;
    }
```